README.md

```diff
@@ -640,6 +641,81 @@ $loginProfile = $osLoginServiceClient->getLoginProfile($formattedName);
 $ composer require google/cloud-oslogin
 ```
 
+## Google Cloud Tasks (Beta)
+- [API Documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs/latest/tasks/cloudtasksclient)
+- [Official Documentation](https://cloud.google.com/cloud-tasks/docs)
+
+#### Preview
+
+```php
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Google\Cloud\Tasks\V2beta2\CloudTasksClient;
+use Google\Cloud\Tasks\V2beta2\LeaseDuration;
+use Google\Cloud\Tasks\V2beta2\PullMessage;
+use Google\Cloud\Tasks\V2beta2\PullTarget;
+use Google\Cloud\Tasks\V2beta2\Queue;
+use Google\Cloud\Tasks\V2beta2\Task;
+use Google\Cloud\Tasks\V2beta2\Task_View;
+use Google\Protobuf\Duration;
+
+$client = new CloudTasksClient();
+
+$project = 'example-project';
+$location = 'us-central1';
+$queue = uniqud('example-queue-');
```

Comment:
This should be `uniqid`. The issue persists on the other sample too.

Reply:
Doh. Fixed